api/src/umigrate/settings.py

```diff
@@ -340,3 +341,12 @@
 
 MEDIA_URL = "/media/"
 MEDIA_ROOT = os.path.join(BASE_DIR, "../media")
+
+SCHEDULER_QUEUE = "{}_default".format(STAGE_ENVIRONMENT)
```

Comment:
We tend to use string interpolation with the $ symbol elsewhere, so we may want to be consistent with that.
`$"{STAGE_ENVIRONMENT}_default"`